apex_launchtest/package.xml

```diff
@@ -1,19 +1,18 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="2">
-  <name>apex_rostest</name>
+  <name>apex_launchtest</name>
```

Comment:
this name `apex_launchtest` suggests to me that we can only test launch files with this framework?

I believe you guys discussed the name change and why it is so?

Reply:
Can you clarify?  "We can only test launch files" as opposed to what? "We can test launch files and . . . "

We removed the ROS deps into apex_launchtest_ros so the tool could be used by things like gazebo, which use launch but not launch_ros.  The structure mirrors that of [ros2 launch](https://github.com/ros2/launch).  If you want to test ROS stuff, the utilities to do so will be in apex_launchtest_ros.

Reply:
In the offline discussion: apex_launchtest launches the launch file(s) which contain nodes/executables under the test.

Reply:
I see - the concern is that the name implies that you only test the launch files themselves and not "This is a tool like rostest (for ros1) but using ros2 style launch and also more new features"